integration_tests/models/schema_tests/schema.yml

```diff
@@ -505,6 +505,20 @@ models:
               value_set: [0.5]
               top_n: 1
               quote_values: false
+              data_type: float
+          - dbt_expectations.expect_column_most_common_value_to_be_in_set:
+              value_set: [0.5, 0, 1]
+              top_n: 2
+              quote_values: false
+              data_type: float
```

Comment:
fyi, Snowflake won't correctly compare a `float` in the db to a value cast to `decimal` (default for the test)